src/DotNet.Testcontainers/Containers/Configurations/Abstractions/HostedServiceConfiguration.cs

```diff
@@ -26,8 +26,6 @@ protected HostedServiceConfiguration(string image, int defaultPort, int port)
 
     public int Port { get; set; }
 
-    public virtual string Hostname { get; set; } = "localhost";
```

Comment:
The removal of this configuration property is a breaking change, so we might need to increment the major version of this library.

Reply:
You're right about that. But I don't think the property was actually used and I don't care about breaking changes. Too much work for such a small project.